examples/flutter_gallery/lib/demo/cupertino/cupertino_navigation_demo.dart

```diff
@@ -240,19 +262,21 @@ class Tab1RowItem extends StatelessWidget {
               ),
               CupertinoButton(
```

Comment:
dartfmt output looks better in this case.

Reply:
agree. It was a mistake before.